lib/fastlyctl/commands/logging.rb

```diff
@@ -0,0 +1,65 @@
+require "fastlyctl/commands/logging/bigquery"
+
+module FastlyCTL
+    class LoggingSubCmd < Thor 
+      namespace :logging
+
+      desc "bigquery <action>", "Setup BigQuery As a logging provider, available actions are create, update, delete, list and show"
+      method_option :service, :aliases => ["-s","--service"], :banner => "Service ID to use", :required => true 
```

Comment:
We should add short hand options (aliases) to more of these to keep it consistent with the rest of the commands.

Reply:
Updated